nixos/modules/services/monitoring/cockpit.nix

```diff
@@ -15,6 +15,8 @@ let
     mkPackageOption
     ;
   settingsFormat = pkgs.formats.ini { };
+  pluginPaths = lib.concatMap (p: p.passthru.cockpitPath) cfg.plugins;
+  cockpitPkg = cfg.package.override { extraSessionPaths = pluginPaths; };
```

Comment:
Personally I disagree with the approach of rebuilding cockpit every time a plugin is added.

Maybe instead, a runtime path like `/etc/cockpit-path` (or another) could be prepended to `default_session_path` which then, at runtime, gets set to the result of a `pkgs.buildEnv` with the plugins?

(While we are here, the session path should probably have Cockpit itself added as well, e.g. using `$out` or `${placeholder "out"}` or similar)